executing/executing.py

```diff
@@ -341,7 +371,7 @@ def executing(cls, frame_or_tb):
                     if TESTING:
                         raise
 
-                if node:
+                if node and stmts is not None:
```

Comment:
This is redundant, I think a cast or an assert would be clearer.

Reply:
Added an assert